### ISSUE TITLE
Avoid placing shader files in `Resources` directories.

### ISSUE THREAD

**Issue by Santarh** [vrm-c/UniVRM#1341]
依存シェーダファイルを `Resources` ディレクトリに置くことをやめたい。

ShaderVariantCollection に揃える。

**Comment by andreiagmu** [comment]
I'm also interested in this update, so that my builds won't compile shader variants that I don't need (I don't use MToon on my project), and to also make my build size a little smaller.

In my tests, I've used a workaround: 
I copied the UniVRM-related packages to my <Project root>\Packages folder, then I renamed all "Resources" folders inside to "Resource". Then, it seems the VRM importer still works normally.

This workaround solves the extra shader variants/build size issue.
But putting the UniVRM packages in that custom Packages folder makes it much less intuitive to update the packages.
